buffer/buffer.go

```diff
@@ -42,6 +42,11 @@ func (b *Buffer) AppendByte(v byte) {
 	b.bs = append(b.bs, v)
 }
 
+// AppendByteV writes a single byte to the Buffer.
+func (b *Buffer) AppendByteV(v ...byte) {
+	b.bs = append(b.bs, v...)
+}
```

Comment:
This should probably be named `AppendBytes` and take a `[]byte`, not a vararg. (Also, the docstring is inaccurate.)

(FYI, there's also Buffer.Write which does the same, while satisfying io.Writer, but there's no problem with also having this method. However, if we have both, maybe Write should call AppendBytes.)